app/va/va_profile/va_profile_client.py

```diff
@@ -1,40 +1,31 @@
 from __future__ import annotations
+from enum import Enum
+from http.client import responses
 
 import iso8601
 import requests
+
+from app.va.identifier import transform_to_fhir_format, OIDS, IdentifierType
 from app.va.va_profile import (
     NoContactInfoException,
     VAProfileNonRetryableException,
     VAProfileRetryableException,
 )
-from app.va.identifier import is_fhir_format, transform_from_fhir_format, transform_to_fhir_format, OIDS, IdentifierType
-from app.va.va_profile.exceptions import VAProfileIDNotFoundException
-from enum import Enum
-from http.client import responses
-from time import monotonic
-from typing import Dict, List, TYPE_CHECKING
+from app.va.va_profile.exceptions import CommunicationItemNotFoundException, VAProfileIDNotFoundException
 
+from typing import Dict, List, TYPE_CHECKING
 
 if TYPE_CHECKING:
     from app.models import RecipientIdentifier
     from va_profile_types import ContactInformation, CommunicationPermissions, Profile, Telephone
 
-EMAIL_TYPE = 'email'
-LETTER_TYPE = 'letter'
-MOBILE_TYPE = 'mobile'
-PUSH_TYPE = 'push'
-SMS_TYPE = 'sms'
 
 VA_NOTIFY_TO_VA_PROFILE_NOTIFICATION_TYPES = {
-    EMAIL_TYPE: 'Email',
-    SMS_TYPE: 'Text',
+    'email': 'Email',
+    'sms': 'Text',
 }
```

Comment:
See [this convo](https://github.com/department-of-veterans-affairs/notification-api/pull/1880/files#r1700644266) in previous PR - this was brought in, in a somewhat hamfisted manner.  Cleaning up